Snowflake.Data/Core/HttpUtil.cs

```diff
@@ -73,8 +74,18 @@ public class HttpClientConfig
     public sealed class HttpUtil
     {
         static internal readonly int MAX_BACKOFF = 16;
+        private static readonly int s_baseBackOffTime = 1;
+        private static readonly int s_baseLoginBackOffTime = 4;

```

Comment:
Why 4? As I see in the doc and in python implementation, first sleep time is 1s?

Reply:
It's based from this [discussion ](https://github.com/snowflakedb/snowflake-sdks-drivers-issues-teamwork/issues/575#issuecomment-1760362338) where base time for login requests is 4